pkg/storkctl/migrationschedule.go

```diff
@@ -93,9 +152,13 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 	createMigrationScheduleCommand.Flags().BoolVarP(&startApplications, "startApplications", "a", false, "Start applications on the destination cluster after migration")
 	createMigrationScheduleCommand.Flags().StringVarP(&preExecRule, "preExecRule", "", "", "Rule to run before executing migration")
 	createMigrationScheduleCommand.Flags().StringVarP(&postExecRule, "postExecRule", "", "", "Rule to run after executing migration")
-	createMigrationScheduleCommand.Flags().StringVarP(&schedulePolicyName, "schedulePolicyName", "s", "default-migration-policy", "Name of the schedule policy to use")
+	createMigrationScheduleCommand.Flags().StringVarP(&schedulePolicyName, "schedulePolicyName", "s", "default-migration-policy", "Name of the schedule policy to use. "+
+		"If you want to create a new interval policy, use interval flag instead.")
 	createMigrationScheduleCommand.Flags().BoolVar(&suspend, "suspend", false, "Flag to denote whether schedule should be suspended on creation")
-
+	createMigrationScheduleCommand.Flags().BoolVar(&autoSuspend, "auto-suspend", true, "If set to true, in the event of a disaster, Stork will automatically suspend the DR migration schedules on your source cluster, "+
+		"allowing you to migrate your application to an active Kubernetes cluster. ")
+	createMigrationScheduleCommand.Flags().IntVarP(&intervalMinutes, "interval", "i", 30, "Specify the periodic interval, in minutes, after which Stork should trigger the operation.")
+	createMigrationScheduleCommand.Flags().StringToStringVar(&annotations, "annotations", map[string]string{}, "Add required annotations to the resource in the format key1=value1,key2=value2,... ")
```

Comment:
If user has to use annotations flag then they will have to input something like this ->
`--annotations openstorage.io/auth-secret-namespace=val1,openstorage.io/auth-secret-name=val2` which seems a bit inconvenient
If in the annotations need to take only these 2 above values , then we could instead add 2 separate flags for them
Something like `--authSecretNamespace val1 --authSecretName val2`

@pp511 @adityadani

Reply:
We should  comma separated annotations to make it future proof.